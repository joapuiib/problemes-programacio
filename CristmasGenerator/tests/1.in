2020 2021
Mariona Marta
Marc Martina
Feliç 2020 Mariona.
Et desitgem que confiïs més en tu mateixa Mariona, que busquis, trobis i aconsegueixis tots els teus objectius, que superis tots els teus límits,  que t'envolti l'amor per tot arreu, i que t'adonis d'una punyetera vegada que ets increïble tal com ets  i  que pots aconseguir tot allò que et proposis.

Fins aviat
Marc
END

