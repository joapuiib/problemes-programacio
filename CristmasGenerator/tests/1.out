Feliç 2021 Marta.
Et desitgem que confiïs més en tu mateixa Marta, que busquis, trobis i aconsegueixis tots els teus objectius, que superis tots els teus límits,  que t'envolti l'amor per tot arreu, i que t'adonis d'una punyetera vegada que ets increïble tal com ets  i  que pots aconseguir tot allò que et proposis.

Fins aviat
Martina
